tests/components/plex/test_config_flow.py

```diff
@@ -47,6 +47,8 @@
 from tests.common import MockConfigEntry
 from tests.typing import ClientSessionGenerator
 
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
```

Comment:
Sadly it seems that `reauth` and `option_flow` tests are relying to a real setup right now.

@jjlawren I think the tests should be adjusted.

For reauth, I think the "triggering" of reauth should be tested in `test_init.py` and the actual flow should be tested in `test_config_flow.py`.
I will attempt to split them tomorrow.

For options flow I haven't quite worked out how they work yet...

Reply:
There's not a reliance on an actual real Plex setup, but a "real" instance object which has been fed mocked `requests` XML payloads.